sessionize/lib/src/commonTest/kotlin/co/touchlab/sessionize/EventModelTest.kt

```diff
@@ -68,4 +79,26 @@ class SessionizeApiMock : SessionizeApi {
         rsvpCalled = true
         return true
     }
+}
+
+class FeedbackApiMock : FeedbackApi {
+
+    var generatingFeedbackDialog:Boolean = false
+    var feedbackError : FeedbackApi.FeedBackError? = null
+
+
+    private var feedbackModel:FeedbackModel = FeedbackModel()
+
+    fun getFeedbackModel(): FeedbackModel {
+        return feedbackModel
+    }
+    override fun generateFeedbackDialog(session: MySessions){
```

Comment:
Tests dont pass. partly because this needs to be `MyPastSessions`, but also seems to be an issue with notificationsapi